src/DocoptNet/DocoptNet.csproj

```diff
@@ -1,15 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net40;netstandard1.5</TargetFrameworks>
+    <TargetFrameworks>net40;netstandard1.5;netstandard2.0</TargetFrameworks>
     <AssemblyName>DocoptNet</AssemblyName>
     <Version>0.6.1.10</Version>
     <AssemblyOriginatorKeyFile>DocoptNet.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly>
     <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
     <PackageId>DocoptNet</PackageId>
     <Copyright>Copyright (c) 2013 Dinh Doan Van Bien, dinh@doanvanbien.com</Copyright>
-    <NetStandardImplicitPackageVersion Condition=" '$(TargetFramework)' == 'netstandard1.5' ">1.6.0</NetStandardImplicitPackageVersion>
```

Comment:
If we aren’t referencing a newer version of the .net standard what is gained by this?